Casks/l/ltspice.rb

```diff
@@ -8,8 +8,8 @@
   homepage "https://www.analog.com/en/design-center/design-tools-and-calculators/ltspice-simulator.html"
 
   livecheck do
-    url :url
-    strategy :extract_plist
+    url :homepage
+    regex(/for\s+MacOS[\s\S]*Version\s+v?(\d+(?:\.\d+)+)/i)
```

Comment:
```suggestion
    regex(/for\s+MacOS.+?Version\s+v?(\d+(?:\.\d+)+)/im)
```

`[\s\S]*` (i.e., whitespace or not-whitespice) seems like it would match anything, so we can simply use `.+`. We also want that part of the regex to be non-greedy, `.+?`, so it will only match the next version after "for MacOS". [A non-greedy match could match subsequent version text and return an incorrect value. For example, if they changed the button order such that the Windows button and version came after the macOS one, the regex would end up matching the Windows version.]

I've also added the multiline (`m`) flag, as it's necessary for `.+?` to match across lines (we only add it when it's necessary):

```
<span class="analog-rounded-button">
    <a href="https://ltspice.analog.com/software/LTspice.pkg" id="d323aa045406444aaf8309ec02cf924d" target="_blank">
Download for MacOS 10.15 and forward                                        </a>
</span>

<span>
Version
    17.2.4
</span>
```

Reply:
> I've also added the multiline (m) flag, as it's necessary for .+? to match across lines

Thanks! I knew there had to be a better solution than `[\s\S]*` to match newline characters.